tests/testthat/helper-server.R

```diff
@@ -62,6 +62,7 @@ hintr_server <- function(n_tries = 10, poll = 0.5, results_dir = tempdir()) {
       TRUE
     }, error = function(e) FALSE)
     if (ok) {
+      withr::defer_parent(process$kill())
```

Comment:
Manually killing this instead of relying on garbage collect I hope will help with memory problems